components/usage/pkg/db/usage.go

```diff
@@ -200,3 +200,33 @@ func GetUsageSummary(ctx context.Context, conn *gorm.DB, attributionId Attributi
 		CreditCentsBalanceAtEnd:   creditCentsBalanceAtStart.Int64 + creditCentsBalanceInPeriod.Int64,
 	}, nil
 }
+
+type Balance struct {
+	AttributionID AttributionID `gorm:"column:attributionId;type:varchar;size:255;" json:"attributionId"`
+	CreditCents   CreditCents   `gorm:"column:creditCents;type:bigint;" json:"creditCents"`
```

Comment:
I may have missed this discussion elsewhere but what is the purpose of the `CreditsCents` type? When we wrote the other reconciler we always stored credit values as floating point values.

Reply:
Basically it stores credits * 100 rounded. This removes precision errors. Here, we use a custom type to represent that it's not credits, but credits * 100, or CreditCents